lessons/06_SC_SCT_normalization.md

```diff
@@ -123,6 +123,24 @@ DimPlot(seurat_phase,
 <img src="../img/pre_phase_pca.png" width="800">
 </p>
 
+> #### Example of when cell cycle phase needs to be regressed out
+> Below are two PCA plots taken from the Seurat vignette dealing with ["Cell-Cycle Scoring and Regression"](https://satijalab.org/seurat/archive/v3.1/cell_cycle_vignette.html).
+>
+> This first plot is similar to what we plotted above, it is a PCA prior to regression to evaluate if the cell cycle is playing a big role in driving PC1 and PC2. 
+> <p align="center">
+> <img src="../img/cell_cycle_not_regressed.png" width="400">
+> </p>
+>
+> Clearly, the cells are separating by cell type in this case, so the vignette suggests using the `ScaleData()` function to regress out this impact.
```

Comment:
@rkhetani do we want to say `ScaleData()`  even though we are addressing the regression using `SCT`?

If you're okay with me changing this I'll go ahead and merge the pull request . Thanks for this!